src/Service/RequestService.php

```diff
@@ -784,6 +798,130 @@ public function proxyHandler(array $data, array $configuration, ?Source $proxy =
 
     }//end proxyHandler()
 
+    public function useRelayRating(array &$config): bool
```

Comment:
missing docblock

Reply:
This is also a function that is prepared for future use, but cannot be used right now as rating is not operating properly yet.